c/examples/Makefile

```diff
@@ -11,7 +11,7 @@
 #
 # $ git clone git@github.com:tskit-dev/tskit.git --recurse-submodules
 #
-# See the documentation (https://tskit.readthedocs.io/en/stable/c-api.html)
+# See the documentation (https://tskit.dev/tskit/docs/latest/c-api.html)
```

Comment:
Maybe keep this one as "stable"? I think the idea was that C developers using this are going to be a long way behind the cutting edge, and are better off seeing the stable docs.

Reply:
Sure. Good call.